web/modules/custom/bmw_cars_module/bmwcars.module

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * Implementing hook_theme().
+ */
+function bwm_cars_module_theme($existing, $type, $theme, $path) {
+
+  return array(
+    'bmwcars' => array(
+      'variables' => array('cars' => array(), 'title' => '')
```

Comment:
ne moras da pises 100x array(), dovoljno je da samo kazes [] , to je ekvivalentno da je niz.